webapp/views/wpt-results.js

```diff
@@ -90,6 +91,14 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
       td.numbers {
         white-space: nowrap;
         color: black;
+        opacity: 1;
+      }
+      td.triage {
+        cursor: pointer;
+        transition: 0.2s;
+      }
+      td.triage:hover {
+        opacity: 0.7;
```

Comment:
```suggestion
        opacity: 0.7;
        box-shadow: 5px 5px 5px;
```
Nit: just to make this a bit more prominent.